soc/infineon/cat1b/cyw20829/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Infineon CAT1B devices
+
+# Series definitions
+config SOC_SERIES_CYW20829
+	select ARM
+	select CPU_HAS_ARM_MPU
+	select CPU_CORTEX_M33
+	select CPU_HAS_FPU
+	select DYNAMIC_INTERRUPTS
+	select SOC_FAMILY_INFINEON_CAT1
```

Comment:
wouldn't it make more sense to have the cat1b family selecting the cat1 family here:
https://github.com/zephyrproject-rtos/zephyr/blob/d59bd090ad4138a3840143b3ffe2cd8ac54f0d40/soc/infineon/cat1b/Kconfig.soc#L9-L10

as:
```
config SOC_FAMILY_INFINEON_CAT1B 
	bool 
	select SOC_FAMILY_INFINEON_CAT1
```